src/test/unit/flight_failsafe_unittest.cc

```diff
@@ -436,7 +436,7 @@ TEST(FlightFailsafeTest, TestFailsafeSwitchModeStage2Land)
 
     // then
     EXPECT_TRUE(failsafeIsActive());
-    EXPECT_TRUE(isArmingDisabled());
+//    EXPECT_TRUE(isArmingDisabled());              // stick induced failsafe allows re-arming in stage1
```

Comment:
We can remove this code, maybe leave in the comment itself.

Reply:
good catch